packages/octane.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Octane < Package
+  description 'The JavaScript Benchmark Suite for the modern web https://developers.google.com/octane/.'
+  homepage 'https://github.com/chromium/octane'
+  version 'v1'
+  source_url 'https://github.com/chromium/octane/archive/v1.tar.gz'
+  source_sha256 '51e9002e8391f65809c19200e44277d0e5a0adf9f7c03af46676cd321f965d68'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'nginx'
```

Comment:
Can we do this via Ruby's webrick, python's built-in webserver, or darkhttpd to make this a smaller install?

Reply:
I suppose that would be worth trying to see if it has any play in the performance factors.  We could also run php as a server as well.

Reply:
Yeah but php is pretty big and most people won't have it installed. We already have ruby installed and most people will have python as well. Just a thought.